MANIFEST.in

```
@@ -18,6 +18,7 @@ include babel.ini
 include docs/requirements.txt
 include LICENSE
 include pytest.ini
+recursive-include *community_theme_template.css
```

```suggestion
include *community_theme_template.css
```
I think it is enough to include since it is not a folder, but a single file